lib/benchmark/performance.rb

```diff
@@ -0,0 +1,70 @@
+# frozen_string_literal: true
+
+module Benchmark
+  class Performance
+    FE = 'frontend'
+    PAGE_PERFORMANCE = 'page_performance'
+
+    # Calls StatsD.measure with the passed benchmarking data. StatsD.measure lets
+    # you benchmark how long the execution of a specific task/method takes.
+    #
+    # @param key [String] A StatsD key. See https://github.com/Shopify/statsd-instrument#statsd-keys
+    # @param duration [Float] Duration of benchmark measurement in milliseconds
+    # @param tags [Array<String>] An array of string tag names
+    # @return [StatsD::Instrument::Metric] The metric that was sent to StatsD
+    # @see https://github.com/Shopify/statsd-instrument#statsdmeasure
+    #
+    def self.track(key, duration, tags:)
+      StatsD.measure(key, duration, tags: tags)
```

Comment:
The `tags` option should work as we are using it for both `increment` and `measure`.  Here is an example for `measure`:

https://github.com/department-of-veterans-affairs/vets-api/blob/master/lib/breakers/statsd_plugin.rb#L41

@wyattwalter - can you please confirm that we are set up to use the `tags` option with `Statsd.measure`, and that the implementation of it will work as expected in this PR?

Reply:
Yep, we support tags. That _should_ work with Prometheus.